os_stub/spdm_cert_verify_callback_sample/spdm_cert_verify_callback_internal.h

```diff
@@ -0,0 +1,60 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2024 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#ifndef __SPDM_CERT_VERIFY_CALLBACK_INTERNAL_H__

```

Comment:
`__SPDM_CERT_VERIFY_CALLBACK_INTERNAL_H__` -> `SPDM_CERT_VERIFY_CALLBACK_INTERNAL_H`

Reply:
Thanks. I have fixed the MACRO.